qcengine/tests/test_procedures.py

```diff
@@ -6,7 +6,7 @@
 from qcelemental.models import OptimizationInput
 
 import qcengine as qcng
-from qcengine.testing import failure_engine, using
+from qcengine.testing import using
```

Comment:
`failure_engine` still needed I think https://github.com/MolSSI/QCEngine/pull/281/files#diff-2307f3b2f47187dcb771c8aa6fb23c8dd62658f566669858c063db3f1a4f5aa9R132

Reply:
Whoops! I'll put it back :)